Source/AGS.API/Graphics/Animations/IAnimationComponent.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System.Threading.Tasks;
 
 namespace AGS.API
-{    
+{
     /// <summary>
     /// An animation container. This gives access to the animation, and allows to start a new animation
     /// which will replace the old animation.
     /// </summary>
+    [RequiredComponent(typeof(ISpriteRenderComponent))]
```

Comment:
Not a big deal, as we're not actually using the RequiredComponent yet, so just and FYI, placing the attribute on the interfaces was a mistake on my part. The component dependencies should be on the implementing class, not on the interface (for example, an alternative implementation of the animation component might not need the sprite component). Once I get to actually using it (in #185 ), I plan on moving all the attributes from the interfaces to the implementations.